Core/src/main/java/com/paypal/android/core/graphql/common/Query.kt

```diff
@@ -0,0 +1,33 @@
+package com.paypal.android.core.graphql.common
+
+import org.json.JSONObject
+
+abstract class Query<T> {
```

Comment:
this could be internal
```suggestion
internal abstract class Query<T> {
```